### DISCUSSION TITLE
Trimming for files already trimmed by Zymo?

### DISCUSSION THREAD

**Discussion by yaaminiv** [RobertsLab/resources#1082] [Q&A] [answered]
I received *C. gigas* gonad methylation data from Zymo. Looking at the [data report](https://github.com/RobertsLab/resources/files/5889545/Epigenetic.Services.Data.Report.Methyl-MaxiSeq.-.Yaamini.Venkataraman_zr3616.pdf), it seems like they already trimmed adapter sequences out and did a 15 bp hard trim on the 5' end.

Is this standard procedure? I've never actually been the one to receive data, so I'm not sure if this is usually what happens. Also, for WGBS data we usually hard trim 10 bp off each end and remove adapters. I'd need to look at a FastQC/MultiQC report for the data I received first, but do I need to modify my trimming protocol based on the trimming conducted by Zymo?

**Comment by kubu4** [comment, marked as answer]
They actually have _not_ trimmed the files. They use that data report template for all of their sequencing. They only perform trimming if bioinformatic services were requested with the order. You'll be able to confirm this by running through FastQC and seeing the existence of the sequencing adapters.

P.S. Please be sure to get these data into the appropriate directory on Nightingales and please add the appropriate info the the Nightingales Google Sheet as soon as you're able. I'm certainly happy to help with this if you don't want to deal with it.

**Reply by yaaminiv** [reply]
Good to know; thanks! 

I need edit access for the Nightingales spreadsheet, but I'll add the files to the correct directory and update the spreadsheet with links
